ACEHAL/HAL.py

```diff
@@ -224,7 +224,7 @@ def _make_ramps(*args, n_stages=20):
                 tau_rel_control.set_tau_rel(tau_rel_cur)
 
                 # set up dynamics for this section of ramp
-                dyn = Langevin(traj_config, dt * ase.units.fs, temperature_K=T_K_cur, friction=T_tau)
+                dyn = Langevin(traj_config, dt_fs * ase.units.fs, temperature_K=T_K_cur, friction=friction)
```

Comment:
I'm confused.  This PR's change appears to just change the name of the argument, but my local copy has `friction = 1.0 / T_tau`.  Did something not get reverted?